jib-core/src/integration-test/java/com/google/cloud/tools/jib/api/JibIntegrationTest.java

```diff
@@ -116,6 +116,21 @@ public void testBasic_dockerDaemonBaseImage()
             targetImageReference.withTag(jibContainer.getDigest().toString()).toString()));
   }
 
+  @Test
+  public void testBasic_dockerDaemonBaseImageToDockerDaemon()
```

Comment:
are both the compressed and uncompressed codepaths now tested?

Reply:
I'll add one for compressed -> docker daemon